android/app/capacitor.build.gradle

```diff
@@ -10,6 +10,7 @@ android {
 apply from: "../capacitor-cordova-android-plugins/cordova.variables.gradle"
 dependencies {
     implementation project(':capacitor-app')
+    implementation project(':capacitor-clipboard')
```

Comment:
when we previously added clipboard copy feature some of the linked dependencies were not tracked in main repo. This isn't really an issue as they are re-populated automatically during build on the server, but added now for tidyness